pom.xml

```diff
@@ -96,7 +96,7 @@
             <version>1.39</version>
         </dependency>
         <dependency>
-            <!-- Requires Java 9+ -->
+            <!-- Required for running with Java 9+ -->
```

Comment:
LGTM. I suppose this is fine, despite the fact we don't anyway really support Java 9, but 8, or 11+ and nothing in between.